dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -18,122 +15,50 @@ const myDataSource = new DataSource({
   database: process.env.TYPEORM_DATABASE,
 });
 
-myDataSource.initialize().then(() => {
-  console.log("Data Source has been initialized!");
-});
+myDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!")})
+  .catch((err) => {
+    console.log("Failed to connect Database", err)
+    appDataSource.destroy();
+  });
```

Comment:
- myDataSource 보다는 `appDataSource` , `mysqlDataSource` 혹은 간결하게 `database`로 변수명을 변경해주세요. “my”라는 용어는 예제 코드에서는 사용할 수 있으나 실제 코드에서는 피해주세요.
- 추가로 `DataSource.initialize()` 함수가 실행되고 나면, then()과 catch()를 사용해서 에러를 핸들링해줄 수 있습니다. 이렇게 하면, initialize() 함수가 실행되어서 database와 실제로 connection을 생성할 때, 성공했는지 실패했는지 여부를 확인할 수 있습니다.

```suggestion
appDataSource.initialize()
  .then(() => {
    console.log("Data Source has been initialized!")})
  .catch((err) => {
    console.log("Failed to connect Database", err)
    appDataSource.destroy();
  });
```

또한 12번 line을 보면 new DataSource()를 통해 만들어진 인스턴스는 `myDataSource`라는 변수에 담아주었는데,  23번 Line을 보면 appDataSource.destroy()를 하고 있습니다. `appDataSource`라는 것은 현재 작성해주신 app.js 어디에도 있지 않기 때문에 동작하지 않을 뿐더러, 실제 .catch() 블럭에 들어오게 되면 에러가 나게 됩니다. 위의 피드백에 맞춰서 모두 수정해주세요-!